tasks/build-doc.js

```diff
@@ -352,6 +352,16 @@ gulp.task('build-guides', ['clean-dist','less'], function () {
                 'step4-define-the-compatibility.md': 'Define the compatibility of your connector'
             }
         },
+        'translation-connection': {
+            title: 'The complete guide to connect Akeneo PIM to your online Translation solution',
+            files: {
+                'introduction.md': 'Introduction',
+                'step1-who-is-your-connector-for.md': 'Who is your connector for?',
+                'step2-understand-akeneo-pim.md': 'Understand Akeneo PIM data',
+                'step3-interact-with-julia.md': 'Interact with Julia',
+                'step4-known-limits.md': 'Known limits'
+            }
+        },
```

Comment:
Can you remove the 'dam-connection-2' guide below? I know it's not part of your PR, but it's totally useless. I can't remember why I added it. Thanks!

Reply:
Sure